lib/common/results.c

```diff
@@ -297,6 +297,14 @@ static const struct pcmk__rc_info {
       "Disabled",
       -pcmk_err_generic,
     },
+    { "pcmk_rc_bad_input",
+      "Bad input value provided",
+      -pcmk_err_generic,
+    },
+    { "pcmk_rc_bad_xml_patch",
+      "Bad XML patch format",
+      -pcmk_err_generic,
+    },
```

Comment:
I'm thinking both of these should map to CRM_EX_DATAERR

Reply:
Added.